src/App.js

```diff
@@ -1,5 +1,7 @@
+/* eslint-disable */
+
 import './App.css';
-import Calculator from './components/calculator';
+import Calculator from './components/Calculator';
```

Comment:
- [ ] The imported **Calculato**r does not match the corresponding file name you're importing. It should be lower case **_c_**calculator and not **_C_**alculator

Reply:
The file name is Calculator.js